### PR TITLE
Create 2021-09-21-incubator-lesson-data-visualization-python.md

### PR DESCRIPTION
**Pull request by kmomar**

### CODE REVIEW

**Comment by kmomar** [comment]
Please Review: @jenna-jordan

File: _posts/2021/09/2021-09-21-incubator-lesson-data-visualization-python.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+------
+
+## About This Series
+
+Incubator Lesson Spotlight is a regular feature in The Carpentries blog and [Carpentries Clippings newsletter][newsletter], highlighting the great work our community is doing to develop new lessons in [The Carpentries Incubator][incubator]. Developers of any lesson in the Incubator are encouraged to submit details about their material for inclusion in the series. If you would like to increase the visibility of your lesson and encourage Carpentries community members to contribute to its ongoing development, [fill in this short form][ils-form].
+
+<!-- link references -->
+[ils-form]: https://forms.gle/cCuLATAEomfdFejs9
+[incubator]: https://github.com/carpentries-incubator/
+[lesson-pages]: <lesson URL from form>
```
Comment (MCMaurer) [inline _posts/2021/09/2021-09-21-incubator-lesson-data-visualization-python.md]: Whoops, I forgot to add the lesson URL here. Should be https://carpentries-incubator.github.io/python-interactive-data-visualizations/

File: _posts/2021/09/2021-09-21-incubator-lesson-data-visualization-python.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+## Lesson Profile
+
+* **Title:** {{page.lesson_title}}
+* **Lesson Pages:** https://carpentries-incubator.github.io/python-interactive-data-visualizations/
+* **Lesson Repository:** https://github.com/carpentries-incubator/python-interactive-data-visualizations
+
+### Learning Objectives
+
+1. How to wrangle data into the right format for creating visualizations using the pandas library (and about Tidy Data)
```
Comment (jenna-jordan) [inline _posts/2021/09/2021-09-21-incubator-lesson-data-visualization-python.md]: 1. Learn how to wrangle data into a tidy format using the pandas library.

File: _posts/2021/09/2021-09-21-incubator-lesson-data-visualization-python.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+2. How to create some basic visualizations using the Plotly Express library
```
Comment (jenna-jordan) [inline _posts/2021/09/2021-09-21-incubator-lesson-data-visualization-python.md]: 2. Learn how to create some basic visualizations using the Plotly Express library.

File: _posts/2021/09/2021-09-21-incubator-lesson-data-visualization-python.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+3. How to create a web app for interacting with the visualizations using the Streamlit library
```
Comment (jenna-jordan) [inline _posts/2021/09/2021-09-21-incubator-lesson-data-visualization-python.md]: 3. Learn how to create a web app using the Streamlit library for sharing the visualizations, as well as interacting with the data and visualizations via widgets.

File: _posts/2021/09/2021-09-21-incubator-lesson-data-visualization-python.md [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+
+### Target Audience
+
+Learners who have recently completed the "Plotting and Programming in Python" lesson (or have some experience with Python and pandas) and want to learn more about data visualizations - and building interactive "dashboards".
```
Comment (jenna-jordan) [inline _posts/2021/09/2021-09-21-incubator-lesson-data-visualization-python.md]: This workshop is appropriate for learners who have recently completed the "Plotting and Programming in Python" lesson (or have some experience with Python and pandas), and  is targeted specifically to those who want to learn more about creating data visualizations and building interactive dashboards. The lesson can be taught in 1 hour (if you skip the exercises and include Episode 1 in the prep) or up to 2 hours (if you include time for students to work through all the exercises and include Episode 1 in the workshop time).

File: _posts/2021/09/2021-09-21-incubator-lesson-data-visualization-python.md [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+
+## Lesson Progress
+
+The lesson is in a finished state - all of the lessons are written and there are exercises included. I have taught the lesson once earlier in its development. It's now ready to be taught by other instructors (but has not yet been taught by other instructors), and I am looking forward to their feedback on the lesson. There are some instructor notes on that extras page. The lesson is currently in "Alpha" stage.
```
Comment (jenna-jordan) [inline _posts/2021/09/2021-09-21-incubator-lesson-data-visualization-python.md]: The lesson is currently in "Alpha" stage - all of the episodes are written, there are exercises included, and there are instructor notes that may be helpful on that extras page. I have taught the lesson once earlier in its development. It is now ready to be taught by other instructors (but has not yet been taught by other instructors), and I am looking forward to everyone's feedback on the lesson.

File: _posts/2021/09/2021-09-21-incubator-lesson-data-visualization-python.md [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+
+## How You Can Help Develop This Lesson
+
+I would like for other instructors to teach the lesson and provide feedback on the experience, especially as regards pacing, clarity, and points where learners struggled.
```
Comment (jenna-jordan) [inline _posts/2021/09/2021-09-21-incubator-lesson-data-visualization-python.md]: I would love for other instructors to teach the lesson and provide feedback on the experience, especially as regards pacing, clarity, and points where learners struggled. I'd also welcome feedback from anyone willing to simply look the lesson over!

File: _posts/2021/09/2021-09-21-incubator-lesson-data-visualization-python.md [hunk 8]
```diff
@@ -0,0 +1,50 @@
...
+
+To contact me directly regarding questions, email me at jenna.jrdn@gmail.com. If there's a problem with the lesson, then submit an issue to the GitHub repo.
```
Comment (jenna-jordan) [inline _posts/2021/09/2021-09-21-incubator-lesson-data-visualization-python.md]: To contact me directly regarding questions about the lesson, you can email me at jenna.jrdn@gmail.com. You can also reach me on Twitter, at [@JennaJrdn](https://twitter.com/JennaJrdn). If there is a problem with the lesson, or you have a suggestion for improvement, then please submit an issue to the GitHub repo!

File: _posts/2021/09/2021-09-21-incubator-lesson-data-visualization-python.md [hunk 9]
```diff
@@ -0,0 +1,50 @@
+---
+layout: page
+authors: ["Jenna Jordan"]
+teaser: "Get involved in the community’s development of a new lesson on the development of interactive data visualizations in Python."
```
Comment (jenna-jordan) [inline _posts/2021/09/2021-09-21-incubator-lesson-data-visualization-python.md]: teaser: "Get involved in the community’s development of a new lesson on how to create interactive data visualizations in Python."

**Review by jenna-jordan — CHANGES_REQUESTED** [review]

**Pull request merged**